htdocs/core/class/html.formprojet.class.php

```diff
@@ -101,14 +100,14 @@ function select_projects($socid=-1, $selected='', $htmlname='projectid', $maxlen
 		}
 		else
 		{
-			$out.=$this->select_projects_list($socid, $selected, $htmlname, $maxlength, $option_only, $show_empty, $discard_closed, $forcefocus, $disabled, 0, $filterkey, 1, $forceaddid, $htmlid, $morecss);
-		}
-		if ($discard_closed)
-		{
-			if (class_exists('Form'))
+			$out.=$this->select_projects_list($socid, $selected, $htmlname, $maxlength, $option_only, $show_empty, $discard_closed, $forcefocus, $disabled, 0, $filterkey, 1, $forceaddid, $htmlid);
```

Comment:
Why did your remove parameter $morecss ?

Reply:
hum, it was a test  I do on an another branch. this haven't to be here... I think I have mixed this dev with a test i did during this development. 

So I close this PR and go back to a new one